saleor/graphql/order/mutations/draft_order_complete.py

```diff
@@ -70,6 +72,17 @@ def validate_order(cls, order):
             )
         return order
 
+    @classmethod
+    def setup_voucher_customer(cls, order):
+        if (
+            order.voucher
+            and order.voucher_code
+            and order.voucher.apply_once_per_customer
+        ):
```

Comment:
We need to assign customer only when `include_draft_order_in_voucher_usage` flag is set to `True`

Reply:
done